mkdocs.yml

```diff
@@ -0,0 +1,74 @@
+# Set strict to true to turn warnings into errors to catch warnings within CI.
```

Comment:
Not sure if we will ever need to use this to generate a guide page, since we are using git-book already

Reply:
if we don't think we need I'll delete all the related files

Reply:
I think we can just use Sphinx to setup something similar to this: 
https://terra-money.github.io/terra.py/index.html

And in the git-book we can just link it: 
https://docs.seinetwork.io/smart-contracts-and-local-development/sei-tool-guide